src/main.ts

```diff
@@ -90,7 +95,11 @@ export default class DataAnalysisPlugin extends Plugin {
 		this.addCommand({
 			id: "builds-corrs",
 			name: "Build Correlations",
-			callback: async () => console.log(this.buildAllCorrelations()),
+			callback: async () => {
+				const corrs = buildAllCorrelations(this.index.data, this.settings.fieldsToCheck);
+				// TODO: don't we want to set the index.corrs to the value here?
+				console.log(corrs);
+			},
```

Comment:
I think I left this cmd in for testing purposes.
I guess it comes down to, do we want a dedicated cmd for building only the corrs? Or should that be handled by Refresh Index? There is also a button on the Correlation View that rebuilds the corrs, so that gets handled there

Reply:
I guess we can just use Refresh Index? If this command is only for debugging or testing purposes, I would rename it so it's clear that users should ignore it.